app/src/main/java/org/fossasia/openevent/dbutils/DbContract.java

```diff
@@ -119,9 +120,9 @@ public DbContract() {
         };
 
         public static final String CREATE_TABLE =
-                "CREATE TABLE " + TABLE_NAME
-                        + " ("
-                        + _ID + " INTEGER PRIMARY KEY,"
+                "CREATE TABLE IF NOT EXISTS " + TABLE_NAME
+                        + " (_id INTEGER PRIMARY KEY,"
+                        // + "_id INTEGER PRIMARY KEY" + COMMA_SEP
```

Comment:
Yeah I was sticking to it as you can see but I ran into a syntax error here so I thought maybe I was doing something wrong here but after I solved the error, I forgot to change this.